### DISCUSSION TITLE
Is the keycloak operator version 20.0.3 Production ready?

### DISCUSSION THREAD

**Discussion by krp2** [keycloak/keycloak#16505] [Archive]
In Keycloak CRD, we see that for pod template, below text is quoted. Wanted to know what is the road map for this? When will it be completely ready? What are the risks if we start using it now.

"You can configure that will be merged with the one configured by default by the operator.
Use at your own risk, we reserve the possibility to remove/change the way any field gets merged in future releases without notice"

**Comment by kereis** [comment]
AFAIK, further major feature implementations are on hold until Store.X is implemented. 

In the mean time, it's possible to use both the new operator and Keycloak Realm Operator at the same time. [Keycloak Realm Operator](https://github.com/keycloak/keycloak-realm-operator) is a fork of the old Keycloak operator but adjusted for the compatibility with the Quarkus distribution. There, you can use CRDs for Client, Realm, Users and ExternalKeycloak (reference for existing Keycloak instances).

**Comment by mithunj43** [comment]
@stianst  any update on this ? we want to use this operator in production so wanted to know is it stable
